index.js

```diff
@@ -0,0 +1,13 @@
+const githubUrl = "https://api.github.com/"
+fetch(githubUrl+"orgs/ieeessitvit/repos")
```

Comment:
Basically using GitHub API **`.updated_at`** method we need to show top 3 repositories recently updated